src/controller/artifact/processor/image/manifest_v2.go

```diff
@@ -65,11 +65,20 @@ func (m *manifestV2Processor) AbstractMetadata(ctx context.Context, artifact *ar
 	artifact.ExtraAttrs["architecture"] = config.Architecture
 	artifact.ExtraAttrs["os"] = config.OS
 	artifact.ExtraAttrs["config"] = config.Config
+
+	author := config.Author
+
 	// if the author is null, try to get it from labels:
 	// https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
-	author := config.Author
+	// https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys
+	authorlist := []string{"org.opencontainers.image.authors", "maintainer"}
```

Comment:
I think we can define this value as `const` for highlight.

Reply:
I don't think we can using keyword `const` to declare an array/list in Golang, since it isn't immutable.